docs/project/changelog.md

```diff
@@ -12,14 +12,18 @@ substitutions:
 
 # Change Log
 
-## Unreleased
+## Version 0.21.1
 
 - New packages: the standard library lzma module {pr}`2939`
 
 - {{ Enhancement }} Pyodide now shows more helpful error messages when
   importing unvendored or removed stdlib modules fails.
   {pr}`2973`
 
+- {{ Enhancement }} `pyodide build` now checks that the correct version of the
```

Comment:
Please also apply changes in the changelog to the main branch.

Reply:
Ok will do.